plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/CustomServerEvaluationStrategy.java

```diff
@@ -99,10 +162,14 @@ public CustomServerEvaluationStrategy(@Nullable @Named("che.docker.ip") String c
 
         // get current ports
         Map<String, List<PortBinding>> ports = containerInfo.getNetworkSettings().getPorts();
-
+        
+        if (isNullOrEmpty(cheDockerCustomExternalTemplate)) {
+            return getExposedPortsToAddressPorts(renderingEvaluation.getExternalAddress(), ports, false);
+        }
+        
         return ports.keySet().stream()
-                    .collect(Collectors.toMap(portKey -> portKey,
-                                              portKey -> renderingEvaluation.render(cheDockerCustomExternalTemplate, portKey)));
+                .collect(Collectors.toMap(portKey -> portKey,
+                                          portKey -> renderingEvaluation.render(cheDockerCustomExternalTemplate, portKey)));
```

Comment:
if there is no change and just formatting, it would be better if these changes are not included in the PR. (if there is a conflict with another branch it can be a harder to merge). Formatting should be done separately to simplify merge IMO.